lib/media/ttml_text_parser.js

```diff
@@ -331,6 +326,20 @@ shaka.media.TtmlTextParser.addStyle_ = function(
       }
     }
   }
+
+  var align = TtmlTextParser.getStyleAttribute_(
+      cueElement, region, styles, 'tts:textAlign');
+  if (align) {
+    if (align == 'center') {
+      // Need to set cue.position to auto to get centered
+      // subtitles
+      cue.align = 'middle';
+      cue.position = 'auto';
```

Comment:
Got you. Let me file some bugs :)
Do I understand the problem with Chrome correctly: cue.align = 'center' is not supported and setting cue.align = 'middle' does not result in text being positioned in the middle of the cue box?